redis_opentracing/__init__.py

```diff
@@ -0,0 +1,149 @@
+from functools import wraps
+
+import redis
+
+g_tracer = None
+g_trace_prefix = None
+g_trace_all_classes = True
+
+def init_tracing(tracer, trace_all_classes=True, prefix='Redis'):
```

Comment:
Can you add doc strings for all the public defs in this file that adhere to [PEP 287](https://www.python.org/dev/peps/pep-0287/)

In particular, clarifying the role of `trace_all_classes` and `prefix` for users would be great.